src/render/tree/mod.rs

```diff
@@ -172,6 +176,10 @@ impl Tree {
         if let Some(ordr) = ctx.sort().map(|s| Order::from((s, ctx.dirs_first()))) {
             ordr.comparator()
                 .map(|func| current_node.sort_children(func));
+        } else if ctx.dirs_first() {
+            Order::from((SortType::None, true))
+                .comparator()
+                .map(|func| current_node.sort_children(func));
         }
```

Comment:
I believe the idiomatic way to do this would be instead to replace the `.map()` with `.map_or_else()` and not have the else branch